ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -1279,7 +1280,7 @@ - (void)decodeRestorableStateWithCoder:(NSCoder *)coder {
     _taskRunUUID = [coder decodeObjectOfClass:[NSUUID class] forKey:_ORKTaskRunUUIDRestoreKey];
     self.showsProgressInNavigationBar = [coder decodeBoolForKey:_ORKShowsProgressInNavigationBarRestoreKey];
     
-    _outputDirectory = [coder decodeObjectOfClass:[NSURL class] forKey:_ORKOutputDirectoryRestoreKey];
+    _outputDirectory = ORKURLFromBookmarkData([coder decodeObjectOfClass:[NSData class] forKey:_ORKOutputDirectoryRestoreKey]);
```

Comment:
Can't use the setter because it's undefined what _hasBeenPresented might be. Should be NO, but ... safer to just ensure directory existence and be explicit about it.